package.json

```diff
@@ -23,5 +23,8 @@
   "dependencies": {
     "concurrently": "^7.6.0",
     "dotenv": "^16.0.3"
+  },
+  "devDependencies": {
```

Comment:
Why do you need webpack?

Reply:
I have now deleted this due to not being needed.